### PR TITLE
Features/step report cross domain unit test

### PR DESCRIPTION
**Pull request by stg2sf**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -129,6 +129,9 @@ Bugfix:
 Changes:
 - refactor test_case, test_suite, test_message_handler module
 
+Features:
```
Comment (BKaDamien) [inline CHANGELOG.md:132]: you don't need to speciy it just respect the commit rules

Comment (BKaDamien) [reply CHANGELOG.md:132]: In addition you have to update the pyprojecttoml with the include

File: setup.py [hunk 2]
```diff
@@ -0,0 +1,96 @@
+##########################################################################
```
Comment (BKaDamien) [inline setup.py:1]: this file is outdated and don't even exist

**Review by BKaDamien — CHANGES_REQUESTED** [review]

**Review by sebastianpfischer — CHANGES_REQUESTED** [review]
So from my understanding, https://github.com/eclipse/kiso-testing/pull/101 does the same no?

**Comment by sebastianpfischer** [comment]
I close the PR because it is covered by #101

**Pull request closed without merging**